tests/flux-v2-image-automation.sh

```diff
@@ -20,6 +20,11 @@ EXCLUSIONS_LIST=(
   apps/idam/idam-web-public/sbox.yaml
   *demo.yaml
   k8s/demo/*
+  apps/*/*/demo-image-policy.yaml
```

Comment:
Could we swap this to wildcard the file name? 
*demo-image-policy.yaml
*perftest-image-policy.yaml
etc